test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Search/BasicSearchTests.cs

```diff
@@ -49,24 +49,11 @@ public async Task GivenResourceWithVariousValues_WhenSearchedWithMultipleParams_
         {
             // Create various resources.
             Patient[] patients = await Client.CreateResourcesAsync<Patient>(
-                p => SetPatientInfo(p, "Seattle", "Robinson"),
-                p => SetPatientInfo(p, "Portland", "Williamas"),
-                p => SetPatientInfo(p, "Seattle", "Jones"));
+                p => SetPatientInfo(p, "Seattle", "Robinson", tag: null),
+                p => SetPatientInfo(p, "Portland", "Williamas", tag: null),
+                p => SetPatientInfo(p, "Seattle", "Jones", tag: null));
```

Comment:
Should we add a tag to these tests? I know that not having tags has caused flaky tests.